workflow_main/Snakefile

```diff
@@ -285,47 +290,21 @@ rule build_full_dataframe:
             output.full_df
         )
 
-rule get_cdc_vocs:
-    output:
-        cdc_list = os.path.join(static_data_folder, "CDC.json")
-    shell:
-        """
-        python3 scripts/get_cdc_vocs.py -o {output.cdc_list}
-        """
-
-rule get_who_vocs:
-    output:
-        who_list = os.path.join(static_data_folder, "WHO.json")
-    shell:
-        """
-        python3 scripts/get_who_vocs.py -o {output.who_list}
-        """
-
-rule get_ecdc_vocs:
-    output:
-        ecdc_list = os.path.join(static_data_folder, "ECDC.json")
-    shell:
-        """
-        python3 scripts/get_ecdc_vocs.py -o {output.ecdc_list}
-        """
-
-rule get_phe_vocs:
-    output:
-        phe_list = os.path.join(static_data_folder, "PHE.json")
-    shell:
-        """
-        python3 scripts/get_phe_vocs.py -o {output.phe_list}
-        """
 
 rule update_vocs:
     input:
-        cdc_list = rules.get_cdc_vocs.output.cdc_list,
-        who_list = rules.get_who_vocs.output.who_list,
-        ecdc_list = rules.get_ecdc_vocs.output.ecdc_list,
-        phe_list = rules.get_phe_vocs.output.phe_list,
+        cdc_list = os.path.join(static_data_folder, "CDC.json"),
+        who_list = os.path.join(static_data_folder, "WHO.json"),
+        ecdc_list = os.path.join(static_data_folder, "ECDC.json"),
+        phe_list = os.path.join(static_data_folder, "PHE.json"),
```

Comment:
This is not the correct way to write snakemake - these are not input files that are *required* to generate the output for this rule. Instead they are intermediate files that are generated by this rule. This scraping job has zero dependencies, so there is no need for the entire input block

You can 1) hardcode the file paths in the `shell` directive, 2) declare them as python variables in the Snakemake file, and then inject them into the `shell` string with `format()`, or 3) declare them in a separate `parameters` block for this rule

Reply:
Actually, best option is to define them as additional outputs for the job – even if we never reference the individual agency files later